src/applications/mhv-medical-records/reducers/labsAndTests.js

```diff
@@ -76,21 +117,24 @@ const convertChemHemRecord = record => {
   const results = record.contained.filter(
     recordItem => recordItem.resourceType === 'Observation',
   );
+  const serviceRequest = extractContainedResource(
+    record,
+    record.basedOn[0]?.reference,
```

Comment:
We should verify that this array has elements before referencing one.

Reply:
Definitely!